shavar/views/__init__.py

```diff
@@ -90,18 +103,13 @@ def format_downloads(request, resp_payload):
     for lname, ldata in resp_payload['lists'].iteritems():
         body += "i:%s\n" % lname
 
-        # Not publishing deltas for this list?  Delete all previous chunks to
-        # make way for the new corpus
-        if _setting(request, lname, 'not_publishing_deltas'):
-            if (len(ldata['adds']) != 1
-                    or len(ldata['subs']) != 0):
-                logger.error("Configuration error!  Mismatch between "
-                             "{lname}'s configuration has "
-                             "'not_publishing_deltas' enabled but its data"
-                             "file has more than one chunk to serve.")
-                raise HTTPInternalServerError()
-            number = ldata['adds'][0].number
-            body += "ad:1-{number}\n".format(number=number - 1)
+        # Chunk deletion commands come first
+        if 'adddels' in ldata:
+            dels = ','.join(['%d' % num for num in ldata['adddels']])
+            body += 'ad:{0}\n'.format(dels)
```

Comment:
Weird mix of old-style and new-style string formatting here; suggest using `.format` consistently